File: tests/fixtures/source_import/enterprise/java/Math.java
package demo;

public final class Math {
    public static long add(long left, long right) {
        long total = left + right;
        return total;
    }

    public static void run() {
        System.out.println(add(2, 3));
    }

    public static long unsupported(long[] values) {
        return values.length;
    }
}

File: tests/fixtures/source_import/enterprise/csharp/Math.cs
using System;

namespace Demo {
    public static class Math {
        public static long Add(long left, long right) {
            long total = left + right;
            return total;
        }

        public static void run() {
            Console.WriteLine(Add(2, 3));
        }

        public static long Unsupported(long[] values) {
            return values.Length;
        }
    }
}
